src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1382,8 +1382,10 @@ public void buildEnvironment(Run<?, ?> build, java.util.Map<String, String> env)
         } else {
             int count=1;
             for(UserRemoteConfig config:userRemoteConfigs)   {
-                env.put("GIT_URL_"+count, config.getUrl());
-                count++;
+               	if(config.getUrl()!=null){
+		env.put("GIT_URL_"+count, config.getUrl());
+                count++; 
```

Comment:
Leading spaces aligned with preceding line
```suggestion
               	    count++; 
```